AsyncDisplayKit/Layout/ASRelativePositionLayoutSpec.h

```diff
@@ -0,0 +1,57 @@
+//
+//  ASRelativePositionLayoutSpec.h
+//  Pods
+//
+//  Created by Samuel Stow on 12/31/15.
+//
+//
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+/** How the child is positioned within the spec. */
+typedef NS_OPTIONS(NSUInteger, ASRelativePositionLayoutSpecPosition) {
+    /** The child is positioned at point 0 relatively to the layout axis (ie left / top most) */
+    ASRelativePositionLayoutSpecPositionZero = 0,
```

Comment:
How about `ASRelativePositionLayoutSpecPosition**Start**` and `ASRelativePositionLayoutSpecPosition**End**`. We have been using "start" and "end" for similar purposes, for example in `ASStackLayoutJustifyContent`, `ASStackLayoutAlignItems`, etc.

P/S: I've just realized that we should use the same convention for `ASHorizontalAlignment` and `ASVerticalAlignment`. Will make an issue for it.